io.github.marco_calautti.DeltaPatcher.yml

```diff
@@ -0,0 +1,27 @@
+id: io.github.marco_calautti.DeltaPatcher
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: DeltaPatcher
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
+  - --device=dri
+  - --share=ipc
+
+modules:
+  - name: DeltaPatcher
+    buildsystem: cmake-ninja
+    post-install:
+      - install -Dm755 app/DeltaPatcher $FLATPAK_DEST/bin/DeltaPatcher
```

Comment:
I don't understand why this is necessary.

Reply:
Do you mean the permissions, or the install? For the install, it just uses the cmake's install to guarantee the dependencies are in the right place. Is there a better way to do it?

Reply:
The install rule already install it, so why doing it again?

Reply:
Ha, I thought it was not done by default. Let me remove it.

Reply:
I actually thought the build would fail because it wouldn't install a .desktop, icon or appstream file... and let the build fail.